content/6-form-input/3-radio/emberPolaris/pick-pill.gjs

```diff
@@ -0,0 +1,32 @@
+import Component from "@glimmer/component";
```

Comment:
hm, radio button demos should probably be added to https://tutorial.glimdown.com 😅